README.md

```diff
@@ -47,23 +47,41 @@ KubeEdge is composed of the following components:
 
 ## Roadmap
 
-### Release 1.0
+### Release 1.0 onwards:
+
 KubeEdge will provide the fundamental infrastructure and basic functionality for IOT/Edge workloads. This includes: 
-- An open source implementation of the cloud and edge parts.
-- Kubernetes application deployment through kubectl from Cloud to Edge nodes.
-- Kubernetes configmap and secret deployment through kubectl from Cloud to Edge nodes and their applications.
-- Bi-directional multiplexed network communication between Cloud and Edge nodes.
-- Kubernetes Pod and Node status querying with kubectl at Cloud with data collected/reported from the Edge.
-- Edge node autonomy when disconnected, and automatic post-reconnection recovery to the Cloud.
-- Device twin and MQTT protocol for communication between IOT devices and Edge nodes.
-
-### Release 2.0 and the Future
+
 - Istio-based service mesh across Edge and Cloud where micro-services can communicate freely in the mesh.
 - Enhance performance and reliability of KubeEdge infrastructure.
 - Enable function as a service at the Edge.
 - Support more types of device protocols to Edge nodes such as AMQP, BlueTooth, ZigBee, etc.
 - Evaluate and enable much larger scale Edge clusters with thousands of Edge nodes and millions of devices.
 - Enable intelligent scheduling of applications to large scale Edge clusters.
+- Data management/analytics framework.
+
+**Release 0.3** - Plan to Release on 3 May 2019:
```

Comment:
31st May, 3rdMay we plan for a beta release.

Reply:
Done